lib/clib/LemmaBucket_C.thy

```diff
@@ -6,13 +6,13 @@
 
 theory LemmaBucket_C
 imports
-  "Lib.Lib"
-  "Word_Lib.WordSetup"
+  Lib.CLib
   CParser.TypHeapLib
-  "CParser.ArrayAssertion"
 begin
 
-declare word_neq_0_conv [simp del]
+context
+notes word_neq_0_conv [simp del]
+begin
```

Comment:
how many of the proofs actually need this? if only one or two, might've made sense to note/supply it there

Reply:
I could have a look at that. I initially wanted to not add too much noise, but there might not actually be any.

Reply:
Done. Disturbance was indeed minimal.